queries/staking/utils.ts

```diff
@@ -23,10 +62,19 @@ export function getEpochDetails(networkId: number, epoch: number) {
 	};
 }
 
-export function getStakingApy(totalStakedBalance: number, weekCounter: number) {
-	const supplyRate = 1 - DECAY_RATE;
-	const initialWeeklySupply = INITIAL_WEEKLY_SUPPLY;
-	const startWeeklySupply = initialWeeklySupply * supplyRate ** weekCounter;
-	const yearlyRewards = (startWeeklySupply * (1 - supplyRate ** 52)) / (1 - supplyRate);
-	return totalStakedBalance > 0 ? (yearlyRewards * STAKING_REWARDS_RATIO) / totalStakedBalance : 0;
+export function getApy(totalStakedBalance: number, weekCounter: number) {
+	const startWeeklySupply = wei(INITIAL_WEEKLY_SUPPLY).mul(SUPPLY_RATE.pow(weekCounter));
+	const yearlyRewards = startWeeklySupply.mul(wei(1).sub(SUPPLY_RATE.pow(52))).div(wei(DECAY_RATE));
+	return wei(totalStakedBalance).gt(0)
+		? yearlyRewards.mul(wei(STAKING_REWARDS_RATIO)).div(wei(totalStakedBalance))
+		: zeroBN;
+}
+
+export function cobbDouglas(feesPaid: number, stakedBalance: number) {
+	// pow(n), n must be a integer
+	return feesPaid ** 0.7 * stakedBalance ** 0.3;
```

Comment:
Same here would be good to update to Wei so we remain consistent with that

Reply:
Thanks for catching it. This function is supposed to be removed as we will read the score directly from the ipfs later.

Still curious how to re-write this to Wei. Is this the right way to do it `return wei(Math.pow(feesPaid, 0.7)).mul(wei(Math.pow(stakedBalance, 0.3)))`?

Reply:
The function has been removed in the latest commit.